electron/legendary/library.ts

```diff
@@ -143,7 +143,6 @@ export class LegendaryLibrary {
 
     if (fullRefresh) {
       try {
-        logInfo('Refreshing Epic Games...', LogPrefix.Legendary)
```

Comment:
The `refresh` function called right after this line also prints this to the logs, it gave the impression that the epic library was getting refreshed twice when it didn't.